Labcoat resprite + medical labcoat

Christasmurf opened the pull request:
## What Does This PR Do

Resprites all existing labcoats
Adds new medical labcoat for doctors
Adds default white labcoat to medical, science and training clothing vendors
Adds staff utility vest to training clothing vendor

## Why It's Good For The Game

Soul removal
NCT should have a labcoat or hazard vest for the appropriate workplaces

## Images of changes

Before:
![image](https://github.com/user-attachments/assets/61376e3f-d8d5-4c32-beca-a8d55fa16687)

After:
![image](https://github.com/user-attachments/assets/83879de3-32b9-42b9-ac6c-888267683141)

![image](https://github.com/user-attachments/assets/520c8558-ec0f-43c6-929f-1426aa84aec4)


## Testing

Above images

<hr>

### Declaration

- [x] I confirm that I either do not require [pre-approval](https://github.com/ParadiseSS13/Paradise/blob/master/docs/CODE_OF_CONDUCT.md#types-of-changes-that-need-approval) for this PR, or I have obtained such approval and have included a screenshot to demonstrate this below.
  <!-- Replace the box with [x] to mark as complete. -->
  <!-- Ensure there are no spaces between the x and the square brackets [] else this will not work properly. -->
<hr>

## Changelog

:cl:
add: Added medical labcoat
imageadd: Resprited all the labcoats
/:cl:

PopeDaveThe3th reviewed (approved):
soul termination

YoureNotTheSharpest commented:
I think the labcoats should cover legs similar to how the security great coat does

Christasmurf commented:
> I think the labcoats should cover legs similar to how the security great coat does

The bottom of the labcoat covers the thighs down to the knees, I didn't send them all the way to the ankles because I wanted to see even the tiniest part of the jumpsuit underneath when it is closed. Purely personal preference.



> I think I prefer the old sprites over these new sprites? The on-ground sprites look pretty out of place compared to a lot of the other items currently in game aswell.

Item sprite perspective is already present on a couple of items (such as the bomber jackets or hazard vests) and is intended for further future resprites as well

> Some of the new labcoat sprites look better than the old ones (for example, the CMO coat, because why is it such a bright blue? It doesn't fit with the mostly-white color scheme of the rest of medbay), but most of the old ones feel like they generally look better on spaceman than the new sprites, and I believe that having the labcoat button/combine in the middle rather than to the left may look better? (I'm not sure how they WOULD actually look in the middle, it's possible it could look really clunky and weird.)

Already put the split in the middle on the closed ones while testing out designs - Didn't look great, especially for the collar shape

> My general idea that I'm trying to get at here is that the new labcoat sprites don't feel like they mesh well with the current clothing hardly at all, and would look really out of place when combined with most of the clothes (this is a maybe, I would probably need to see this in-game more combined with more clothes), and I like the current labcoat look enough that I would prefer to not see them replaced.

Would probably be less out-of-place if I had done all the jumpsuits to accompany, however that is certainly too much work for one PR (Two entire departments when I'm doing them generally 1 job at a time). Additionally, personal preference.

YoureNotTheSharpest commented:
> > I think the labcoats should cover legs similar to how the security great coat does
> 
> The bottom of the labcoat covers the thighs down to the knees, I didn't send them all the way to the ankles because I wanted to see even the tiniest part of the jumpsuit underneath when it is closed. Purely personal preference.

OH no no I meant armor wise, I recall that the labcoats have biological armor and if they are going to extend to the knees like the greatcoat, they might as well have the greatcoats leg armor properties.

Christasmurf commented:
> > > I think the labcoats should cover legs similar to how the security great coat does
> > 
> > 
> > The bottom of the labcoat covers the thighs down to the knees, I didn't send them all the way to the ankles because I wanted to see even the tiniest part of the jumpsuit underneath when it is closed. Purely personal preference.
> 
> OH no no I meant armor wise, I recall that the labcoats have biological armor and if they are going to extend to the knees like the greatcoat, they might as well have the greatcoats leg armor properties.

Oh, in that case - That's for another PR. I don't want to mess with armour values or balancing in this PR.

ExusA reviewed (approved):
Hell yeah :D

Ahzpy commented:
While I like all of these new sprites, I do have a couple complaints.

Firstly, I don't think there needs to be a medical labcoat. It looks very similar to the CMO's, and medical doctors are the only instance I've seen the default labcoat actually be used. I think it works fine for their loadout.

Second, all hydroponics themed items have a mix of vibrant green and dark blue. I worry that the dark green you used will clash with the existing jumpsuit and bandana, and there's a complete lack of dark blue. It could use some on the shoulders, or the entire coat could be dark blue with a green accent, or vice versa.

Other than that, amazing sprite work, I'm a geek for long labcoats.

Christasmurf commented:
> Firstly, I don't think there needs to be a medical labcoat. It looks very similar to the CMO's, and medical doctors are the only instance I've seen the default labcoat actually be used. I think it works fine for their loadout.

I believe the labcoats are just distinct enough, and I may end up tweaking them further alongside department jumpsuits.

Due to this change I added the labcoat to a couple of vendors (Scidrobe, traindrobe, it is still in the medidrobe) just in case people would prefer to use it over the new default. I was also debating adding it as a loadout option for all roles that have labcoats, if that would be something people want.

> Second, all hydroponics themed items have a mix of vibrant green and dark blue. I worry that the dark green and bright white you used will clash with the existing jumpsuit and bandana, and there's a complete lack of dark blue. It could use some on the shoulders, or the entire coat could be dark blue with a green accent, or vice versa.

As mentioned above, I will likely end up tweaking this one whenever I get around to doing botany. Current design is based on discussion from spriting channel in the discord from a while ago. At first I tried green with blue accents, but it hurt my eyes to look at so I changed to green with brown accents. Based on feedback from others, I then changed it to the white with green in the PR. I could see about adding some blue to the arms.
